packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt

```diff
@@ -115,12 +115,24 @@ class ExpoImageViewWrapper(context: Context, appContext: AppContext) : ExpoView(
       activeView.setTintColor(value)
     }
 
-  internal var accessible: Boolean = false
+  internal var isFocusableProp: Boolean = false
```

Comment:
It can't be named: `isFocusable` or `focusable` because the `ExpoImageViewWrapper` has those properties already.